### PR TITLE
Convert README to markdown and add Travis CI build status, closes #765

### LINKED ISSUE #765: Convert README to README.md and add Travis CI build status badge
If it is OK to convert `README` to `README.md` we could add a badge that shows the current build status like this:

[Build Status](https://travis-ci.org/Subsurface-divelog/subsurface)

### PR DESCRIPTION
**Pull request by probonopd**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [x] Documentation change
- [ ] Language translation

### Pull request long description:
Convert `README` file to markdown and add Travis CI build status. Fixes #765

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,17 +118,17 @@ looks like this:
 	Body of commit message is a few lines of text, explaining things
 	in more detail, possibly giving some background about the issue
 	being fixed, etc etc.
-
+
```
Comment (neolit123) [inline README.md]: stray whitespace

Comment (neolit123) [reply README.md]: more bellow.

**Review by neolit123 — CHANGES_REQUESTED** [review]
only a couple of whitespace changes requested.
but i will leave this to @dirkhh.

personally i don't mind this. it makes the repo more 'modern'.

File: README.md (hunk 1 above)
Comment (probonopd) [reply README.md]: Should be addressed in the meantime.

**Review by probonopd — COMMENTED** [review]
Please check latest commit.

**Comment by dirkhh** [comment]
I like it. Thanks @probonopd

**Comment by sfuchs79** [comment]
After renaming the README to README.md we need to change the file name also in some makefiles and maybe some other places. Recognized this because currently creating the Windows NSI installer fails because of this.

CMake Error at cmake_install.cmake:51 (file):
  file INSTALL cannot find
  "/home/stefan/Entwicklung/Subsurface/subsurface/README".

**Comment by probonopd** [comment]
Uh, sorry @sfuchs79 for having broken this. I am not a Windows user, so would appreciate if you could fix it.

**Comment by sfuchs79** [comment]
I guess this is not a pure Windows thing because the README file is also mentioned in a script and some other file :-)
I'll try to catch all occurrences and sent a PR but @dirkhh has to review this carefully.

**Comment by sfuchs79** [comment]
-> #772

**Comment by dirkhh** [comment]
One of the things we SHOULD do is get Travis to also build a Windows installer...
I had started working on doing a Mac build on Travis when they started to offer that option but never got it to work and frankly gave up - but this would be really cool to have.
Not sure if you'd be interested to experiment with that, @sfuchs79

**Comment by sfuchs79** [comment]
@dirkhh 
I for sure agree that this would be very nice.
But the best thing I can offer at the moment is that I put it at the bottom of my Subsurface OIL. And this list is pretty long at the moment.
One small thing I can offer immediately: If you create an issue for this I can add there s.th. nice I found on the web: MXE is providing some instructions and an example project for MXE Travis integration.

**Comment by dirkhh** [comment]
-> #775

**Pull request merged**